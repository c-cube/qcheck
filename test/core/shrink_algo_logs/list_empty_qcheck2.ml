open QCheck2

let lists_are_empty_issue_64 =
  Test.make ~name:"lists are empty"
    Gen.(list small_int) (Log.shrinks Print.(list int) @@ fun xs -> xs = [])
;;
try Test.check_exn lists_are_empty_issue_64 with Test.Test_fail _ -> ()
