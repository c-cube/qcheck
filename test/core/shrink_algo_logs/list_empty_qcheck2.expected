fails [9; 3; 94; 3]
holds []
fails [7; 5]
holds []
fails [1]
holds []
fails [0]
